recipes/logrotate.rb

```diff
@@ -16,10 +16,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 include_recipe 'logrotate'
 
+Chef::DSL::Recipe.include(Nginx::Cookbook::Helpers)
+
+log_dir  = nginx_log_dir
+pid_file = nginx_pid_file
```

Comment:
Why can't these be used directly?

Reply:
@ramereth When used in the custom resource, It leads to an undefined method error. Not sure exactly why, seems like some kind of scoping mismatch. Might be related to [this commit](https://github.com/sous-chefs/nginx/commit/0c9a3a445c4db3269ce878e15d350266c52d45d1).

Reply:
If you change it to use `Chef::DSL::Universal.include(Nginx::Cookbook::Helpers)`, it seems to work and pass cookstyle.